src/debug/ee/debugger.cpp

```diff
@@ -3157,7 +3157,7 @@ HRESULT Debugger::GetILToNativeMappingIntoArrays(
 
     // Get the JIT info by functionId.
 
-    DebuggerJitInfo * pDJI = GetLatestJitInfoFromMethodDesc(pMD);
+    DebuggerJitInfo * pDJI = GetJitInfoFromAddr(pCode);
```

Comment:
Converting code address back to MethodDesc is not cheap operation. It would be better to use GetJitInfo overload that takes both code address and MethodDesc to avoid this lookup.